library/std/src/ffi/c_str.rs

```diff
@@ -1249,7 +1249,7 @@ impl CStr {
         unsafe { &*(bytes as *const [u8] as *const CStr) }
     }
 
-    /// Returns the inner pointer to this C string.
+    /// Returns the inner pointer of this `CStr`.
```

Comment:
While a `CString` *contains* a pointer, a `CStr` is just the bytes of the string itself. It's the `&CStr` that contains/is the pointer. So 'inner pointer' probably isn't the best term here.

Reply:
How about "Returns a raw pointer to the `CStr`"?

Reply:
> How about "Returns a raw pointer to the `CStr`"?

That sounds like it returns a `*const CStr`. For [`str::as_ptr()`](https://doc.rust-lang.org/stable/std/primitive.str.html#method.as_ptr) we just document *"Converts a string slice to a raw pointer."* That exact text might not work for `CStr`, since there's still plans to make `&CStr` a thin pointer (so, not a slice). But maybe just "Returns a raw pointer to the string" or "[..] to the start of the string" could work.

Reply:
I've changed to "Returns a raw pointer to the start of the string."